packages/template-typescript-minimal/app/pages/home.tsx

```diff
@@ -4,171 +4,96 @@
 
             <div className="loading-screen">
-                <div className="panel title">
-                    <h1>
-                        Typescript
-                        <br />
-                        Support!
-                    </h1>
+                <div className="title">
+                    <h1>Welcome!</h1>
                 </div>
-                <div className="panel">
+                <div>
                     <div className="divider"></div>
                 </div>
-                <div className="panel">
-                    <p style={{width: '300px'}} className="fade-in fade-in-0">
-                        <>
-                            <b>This page is written in Typescript</b>
-                            <br />
-                            <br />
-                            React query works if this is not empty: &quot;{query.data}&quot;
-                            <br />
-                            <br />
-                            Server-side getProps works if this is a valid expression: &quot;5 times
-                            7 is {value}
-                            &quot;
-                            <br />
-                            <br />
-                            Client-side JS works if this counter increments: {counter}
-                            <br />
-                            <br />
-                            Application Extensions work if you see a list below:
-                            <br />
-                            <ul>
-                                {applicationExtensions?.map((extension, index) => (
+                <div>
+                    <div style={{width: '300px'}}>
+                        <p>
+                            <a
+                                href="https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/developer-workflow.html"
+                                className="button"
+                            >
+                                Get started
+                            </a>
+                        </p>
+
+                        <p>You have installed the following application extensions:</p>
+                        <ul>
+                            {applicationExtensions.length > 0 ? (
+                                applicationExtensions?.map((extension, index) => (
                                     <li key={index}>{extension.getName()} Extension</li>
-                                ))}
-                            </ul>
-                            <b>You can mix-and-match JS and TS</b>
-                            <br />
-                            <br />
-                            <HelloJS />
-                            &nbsp;
-                            <HelloTS message="it works!" />
-                        </>
-                    </p>
+                                ))
+                            ) : (
+                                <li>None</li>
```

Comment:
This "none" is very computer sounding. Maybe we can make that look a little better. Like:

If you have extensions installed:

```
You have 2 extensions installed.
    - Chakra Storefront
    - Chakra Store Locator
```
If you don't have extensions installed:

```
You currently do not have any extensions installed.
```

Reply:
This by no means is what we'll have in the end. But it's a little cleaner. It would be nice to have friendly names like my examples above, but using the package name is ok for now.